spec/components/schemas/Scheduling/DueTimeShiftInstruction.yaml

```
@@ -0,0 +1,18 @@
+type: object
+description: |
+  The calculation instruction of due time.
+  This is used in conjunction with the **billing anchor** to calculate due time of invoice.
+  The chronology of due time shift is always **after**.
```

If we can rename the billingAnchorInstruction, then this may make more sense like this:
```suggestion
  The chronology of due time shift is always **after** the issuance of the invoice.
```